dll/win32/browseui/shellbars/CBandSiteMenu.cpp

```diff
@@ -97,8 +97,17 @@ HRESULT CBandSiteMenu::_CreateMenuPart()
 
         /* Get the band name */
         StringCchPrintfW(wRegKey, MAX_PATH, L"CLSID\\%s", wszBandGUID);
-        dwDataSize = MAX_PATH;
-        SHGetValue(HKEY_CLASSES_ROOT, wRegKey, NULL, NULL, wszBandName, &dwDataSize);
+        HKEY hKey;
+        if (RegOpenKeyExW(HKEY_CLASSES_ROOT, wRegKey, 0, KEY_READ, &hKey) == ERROR_SUCCESS)
```

Comment:
Note: one alternative would be to keep the SHGetValue call(s) and use SHLoadIndirectString too.
Also, observe that we have in our codebase:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=SHLoadRegUIStringW
"FIXME: Use shlwapi!SHLoadRegUIStringW instead when it is fully implemented",
so is it fine, now? or is it still missing something?

Reply:
Our `shlwapi!SHLoadRegUIStringW` doesn't expand the environment variables.

Reply:
Ah this was the reason :)